docs/ide/editorconfig-code-style-settings-reference.md

```diff
@@ -0,0 +1,896 @@
+---
+title: ".NET Code Style Settings For Editorconfig | Microsoft Docs"
+uid: editorconfig-code-style-settings-reference.md
+ms.custom: ""
+ms.date: "12/07/2016"
+ms.reviewer: ""
+ms.suite: ""
+ms.tgt_pltfrm: ""
+ms.topic: "article"
+dev_langs:
+  - "VB"
+  - "CSharp"
+helpviewer_keywords:
+  - "editor"  
+ms.assetid:
+caps.latest.revision: 01
+author: "kaseyu"
+ms.author: "kaseyu"
+manager: "davidcsa"
+translation.priority.ht:
+  - "cs-cz"
+  - "de-de"
+  - "es-es"
+  - "fr-fr"
+  - "it-it"
+  - "ja-jp"
+  - "ko-kr"
+  - "pl-pl"
+  - "pt-br"
+  - "ru-ru"
+  - "tr-tr"
+  - "zh-cn"
+  - "zh-tw"
+---
+
+# .NET Code Style Settings For Editorconfig
+
+## Possible values
+
+`options_name = false|true : none|suggestion|warning|error`
+
+For code style option you must specify **true** (prefer this option) or **false** (do not prefer this option) a colon (`:`) and a severity (`none`, `suggestion`, `warning`, or `error`). Severity means the level of enforcement for that style you want in your code base
+
+Severity | effect
+------------ | -------------
+none | Do not show anything to the user when this style is not being followed, however code generation features will generate in this style 
+suggestion | When this style is not being followed, show it to the user as a suggestion (underlying dots on the first two characters)
+warning | When this style is not being followed, show a compiler warning 
+error | When this style is not being followed, show a compiler error
+
+## "This." and "Me." Qualification
+
+### Prefer "this." and "Me." for field qualification
+#### Option Name:
+```
+dotnet_style_qualification_for_field
+```
+#### Applicable Languages: C# and Visual Basic
+
+#### Description: 
+
+**true:** Prefer the all non-static fields used in non-static methods to be prefaced with `this` in C# or `Me` in Visual Basic.
+
+C#:
+```C#

```

Comment:
Throughout this topic, all instances of "C#" as the language tag in code snippets should be changed to "cs".  E.g. ```C# would become ```cs.